### PR TITLE
Merging aviyan-dev to main

### PR DESCRIPTION
**Pull request by aveeyan**

### CODE REVIEW

File: mezcla/examples/hugging_face_speechrec.py
Comment (tomasohara) [inline mezcla/examples/hugging_face_speechrec.py:1]: This breaks the dynamic loading of torch: that's why init_torch_etc was added.

File: mezcla/lm/test.arpa
Comment (tomasohara) [inline mezcla/lm/test.arpa:1]: Put this under tests/resources: in general try to avoid special case directories like lm

File: mezcla/samples/pima-indians-diabetes.csv
Comment (tomasohara) [inline mezcla/samples/pima-indians-diabetes.csv:1]: Why was this changed?

File: mezcla/tests/misc_test/revised_checkout_mezcla_aviyan_dev.ipynb
Comment (tomasohara) [inline mezcla/tests/misc_test/revised_checkout_mezcla_aviyan_dev.ipynb:1]: Best to use mezcla/tests/jupyter, as misc_tests too vague.

File: mezcla/tests/test_audio.py
Comment (tomasohara) [inline mezcla/tests/test_audio.py:1]: You can test for a subset of words extracted from the audio 
    system.intersection("testing one two three".split()), 
                                    result_speech)
Do a search for intersection or differente in the test source for other examples.

File: mezcla/tests/test_extract_document_text.py
Comment (tomasohara) [inline mezcla/tests/test_extract_document_text.py:1]: Best to minimize extraneous check-in's: this is not related to the audo test.

File: requirements.txt
Comment (tomasohara) [inline requirements.txt:1]: Make all the audio imports optional except for Hugging Face.

**Review by tomasohara — APPROVED** [review]
Hi, we'll need to do more frequent merges so that it is easier to review.

Some of the older tests shouldn't have been merged with the audio tests. In the future try to limit things to be "on topic"

I approved the merge, but I would like for you to address some of the merge comments such as the placement of testing files.

Merge conflicts will be likely, so check if you need help resolving them.

**Comment by aveeyan** [comment]
My apologies for the delayed merge. I did a hard reset on the branch using aviyan-dev-experimental, where I updated my branch with the main. The conflicts have been resolved accordingly.
